airflow/jobs/triggerer_job_runner.py

```diff
@@ -608,6 +608,8 @@ async def run_trigger(self, trigger_id, trigger):
                 self.log.info("Trigger %s fired: %s", self.triggers[trigger_id]["name"], event)
                 self.triggers[trigger_id]["events"] += 1
                 self.events.append((trigger_id, event))
+        except asyncio.CancelledError:
+            self.log.info("Trigger cancelled")
```

Comment:
The issue that this was trying to address was communicating to the user that there was a timeout. If I was a user and saw "Trigger cancelled" I would be confused because I had not cancelled anything and I would have to trace through the code base (if I'm able as a code-y user) to find out that it was actually a timeout.

Reply:
Also do we want to re-raise the error?

Reply:
Hi @o-nikolas , @uranusjr , thanks for your suggestions! I tried to extend the message as "Trigger cancelled due to timeout" and re-raise this error

Reply:
But is it always true that the Cancelled exception means a timeout? Or are there cases where it's actually cancelled by the user or for some other reason?

Reply:
At first, I believed it to be the case. However, upon further consideration, we may come across different scenarios as well. How about we log the exception itself? like

```python
        except asyncio.CancelledError as err:
            self.log.info("Trigger cancelled due to {str(err)}")
```

Reply:
Do we want that log level to be INFO? I think it would be more accurate to log that as an ERROR

Reply:
Sounds good. I was not sure what would be the best for this case. Thus, I kept it INFO, but let me update it now 💪